### ISSUE TITLE
Support for Remix (similar to '@ant-design/nextjs-registry')

### ISSUE THREAD

**Issue by jackytank** [ant-design/ant-design#48213]
### What problem does this feature solve?

There are already support for NextJS App router for using Ant Design by wrapping <AntdRegistry> then it work perfectly, It would be great if Remix have support like this

### What does the proposed API look like?

import { AntdRemixRegistry } from '@ant-design/remix-registry';

**Comment by devbytyler** [comment]
I would also really appreciate this. Been having a very hard getting dynamic themes to work with Remix. So far, I've determined that it's not possible. Even just some documentation on the subject would be nice.

**Comment by jackytank** [comment]
> I would also really appreciate this. Been having a very hard getting dynamic themes to work with Remix. So far, I've determined that it's not possible. Even just some documentation on the subject would be a huge help.

Indeed, I mean in Remix I just copied the whole <Layout> on Ant Design docs to have a dynamic layout but it didn't work in Remix, hope Remix will be supported next
